cmd/credentialUtil.go

```diff
@@ -345,6 +345,9 @@ func logAuthType(ct common.CredentialType, location common.Location, isSource bo
 		name = "Azure AD (Managed Disk)"
 	}
 	message := fmt.Sprintf("Authenticating to %s using %s", resource, name)
+	if ct == common.ECredentialType.Unknown() && location.IsAzure() {
+		message += ", Please authenticate using Microsoft Entra ID (https://aka.ms/AzCopy/AuthZ), use AzCopy login, or append SAS to your Azure URL."
```

Comment:
"append *a* SAS token"

Second note-- Noting whether this is the source or destination can help customers determine the issue.

Reply:
The info message starts with listing if the source or destination is being authenticated, I don't think it's necessary to repeat the information - `INFO: Authenticating to destination using Unknown`.